zebra-state/src/service/non_finalized_state/chain.rs

```diff
@@ -612,3 +778,9 @@ impl PartialEq for Chain {
 }
 
 impl Eq for Chain {}
+
+impl Default for Chain {
+    fn default() -> Self {
+        Self::new(Default::default(), Default::default(), Default::default())
+    }
+}
```

Comment:
It's not clear to me what a default `Chain` actually is.
How is `Chain::default` used?

Can we use a more specific name than `default`?
Or can we document how it should be used?